### ISSUE TITLE
New placement of Understanding link

### ISSUE THREAD

**Issue by DavidMacDonald** [w3c/wcag#742]
Over the years I've been astonished at the number of top professionals in accessibility and those looking at the guidelines who miss the understanding documents for SCs or don't read them. We've had large organizations come to us asking to clarify things that were in the Understanding document. We've had tool manufacturers file issues about success criteria that when I asked them if they read the understanding document they kind of scratched their heads. I don't want to name names and point out exact circumstances, because it could be embarrassing but I think we could make a small change to our UI for the standard that perhaps could solve it. 

To those of us who are close to our documents, it may seem weird that they never clicked the little link that says "Understanding" next to the SC. Its in the little box over to the right with a link to "how to meet". 

What if we do a classic "Learn more SC [short name]" link immediately following the text of the SC. After the last period. 

![image](https://user-images.githubusercontent.com/2365411/57948364-750e6b80-78af-11e9-98ff-e381907276eb.png)

Just a thought.

**Comment by mraccess77** [comment]
For me that proposed "learn more link" blends into the success criteria.  But I agree we need to do something similar such as moving those links and emphasizing the understanding docs.  Because the understanding docs also contain the SC I always go right to them and just skip the WCAG doc itself (personal preference)

**Comment by jake-abma** [comment]
That surely is an issue but I do have another one on top of this for the Understanding docs themselves.

At GAAD I was at a conference and attended an almost 2 hour presentation of WCAG 2.1 with focus on the new SC.  The presenter was a professional from an (well known) accessibility company doing audits.

It was astonishing to see and hear how the new SC were not really well understood. Many corrections were needed to keep the presentation on track, lacking all the nuances we spend so much time on.

After the presentation I had a talk with the presenter and he told me the company as well as many other people in our country discussed the new SC multiple times within a group and this was the result how they understood the SC.

I would like to suggest / propose we add a specific section in ALL the Understanding docs what is IS about and specifically what it is NOT about. This would really, really help lots of people.

2 simpel / not elaborated examples:
------------------------------------------------
**1.3.4 Orientation:**

Is about:
- Only not locking the orientation from within the content

Is **NOT** about:
- No loss of information or functionality
- Preventing horizontal scroll
------------------------------------------------
**1.3.5 Identify Input Purpose**

Is about:
- Only providing the purpose via an attribute
- The attributes may come from different techniques like microformats / autocomplete or others...
- Purely the user using the web page at that moment, not other users or puposes NOT defined in the WCAG list

Is **NOT** about:
- Filling in data so it's easy for the user
- Only about / requiring autocomplete
- All possible names, emails, etc. on a webpage
------------------------------------------------

This really seems very much needed, they had so much nuances "just a bit wrong" and all agreed with each other if there was not someone (in this case me) to tell them otherwise.

Wondering what you all think of this?!

**Comment by jake-abma** [comment]
ps. some (or all) of this info might be processed within the text somehow but still people just do not extract it this way. A "yes it is / no it isn't" kind of list would help so much more.

**Comment by patrickhlauke** [comment]
@jake-abma agreed, i find that there's a lot of misunderstanding out there - oddly, i had to just explain label in name https://twitter.com/patrick_h_lauke/status/1129493494450597889 and the "best practice" note it has, clarifying that of course that's not a normative requirement.

i do think the problem is also some of the wording we landed on in the end...wordsmithed and tweaked over so many iterations that sometimes it unintentionally lost all meaning. or having aspects/implications that are not evident until now when you try to actually apply them (e.g. see the two discussions we're having separately about having values "down to" / "up to" rather than ONLY normatively requiring those exact values - for text spacing and reflow)

**Comment by patrickhlauke** [comment]
(and heck, even for some of the original 2.0 SCs we're still finding that a lot of interpretation/questioning is happening, which is disconcerting)

**Comment by jake-abma** [comment]
Exactly! Just to give some perspective: out of 12 new AA SC I had to correct 11 of them :-) one of them of course label in name...

**Comment by alastc** [comment]
> over so many iterations that sometimes it unintentionally lost all meaning.

I think it's the reverse, it the terms got **_loaded_** with so much nuanced meaning that it is harder to understand.

For David's initial question, I agree the understanding doc should be more prominent, and something I pushed for in 2.1 was putting the understanding doc as first of those two links. Perhaps we need to go further, but I'd like to mark that aspect as 'design', and ask the new W3C person who is starting soon to tackle it. 

For Jake's question, we had a proposal for that a while ago in #395. It’s a slightly different approach, but in either case we’d need to sit down and create 78 versions…

Could we keep conversation on that topic over there, and keep this issue for the understanding-link aspect.

**Comment by patrickhlauke** [comment]
Closing as stale. Feel free to start a new issue about design-related suggestions/discussions for the spec.
